### ISSUE TITLE
Vesktop white screen

### ISSUE THREAD

**Issue by TheMenko** [NixOS/nixpkgs#263603]
### Describe the bug
Vesktop opens up to a white screen. It cannot update from tray to latest version. 
I tried completely removing config files from Vencord  but it doesn't change anything.
Running Vesktop with flag --disable-gpu makes the Vesktop stuck at loading

### Steps To Reproduce
1. Launch Vesktop

### Expected behavior
Launching Vesktop should open up the app normally

### Screenshots
![image](https://github.com/NixOS/nixpkgs/assets/29785008/81f29fe0-beee-42f7-8cec-5dc150c1f319)

### Additional context
Kernel: 6.5.7-xanmod1
Gpu: AMD (mesa 23.1.7)

### Notify maintainers
@getchoo 
@Scrumplex 

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.7-xanmod1, NixOS, 23.05 (Stoat), 23.05.4466.b3ddf9649fda`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.6`
 - channels(root): `"home-manager-23.05.tar.gz, nixos-23.05, unstable"`
 - channels(menko): `"home-manager-23.05.tar.gz, nixpkgs-23.05"`
 - nixpkgs: `/home/menko/.nix-defexpr/channels/nixpkgs`
```

**Comment by TheMenko** [comment]
vencorddesktop command doesn't output any logs. When doing update from the gui, it doesn't show anything.
Only time it shows something is when i try updating from tray icon. The output is the following:
```console
(node:12186) UnhandledPromiseRejectionWarning: Error: EROFS: read-only file system, open '/nix/store/5y3h89c7fsm1aphxj69jhr6cgj7ayh6a-vencord-1.5.7/vencordDesktopMain.js'
(Use `electron --trace-warnings ...` to show where the warning was created)
(node:12186) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

**Comment by Scrumplex** [comment]
This is a problem with Vencord. It is currently broken with the curren versions of Discord (including the web app)

We have to wait for upstream to fix these issues and release a new version of vencord

**Comment by TheMenko** [comment]
Alright, thank you for the fast response! Cheers

**Comment by ChanceHarrison** [comment]
I encountered the same issue, and can confirm it is fixed when the system version of vencord is 1.6.0.
